src/components/experiment-tracking/run-metadata/index.js

```diff
@@ -4,6 +4,10 @@ import { toHumanReadableTime } from '../../../utils/date-utils';
 
 import './run-metadata.css';
 
+// checks if value is an empty string, which is the default value returned
+// by the graphql endpoint for empty values
+const checkEmptyValue = (value) => (value !== '' ? value : '-');
```

Comment:
Nit: a check in my head returns a boolean. This has side effect, so I'd go for something like `sanitiseEmptyValue`. Feel free to ignore this comment though.

Reply:
Thanks both! I agree with the suggestion of the function naming.

I added the comment above particularly to provide clarity on the format that is returned from the graphql endpoint ( i.e I am checking for an empty string here given the current endpoint setup, and not checking for null or undefined values). I initially had the doubt to check for those other 2 situations but confirmed with the endpoint, and think this piece of information would be important in the assumption for the check executed in this function.